app/jobs/pusher_high_priority_count_changed_job.rb

```diff
@@ -0,0 +1,20 @@
+class PusherHighPriorityCountChangedJob < ApplicationJob
+  queue_as :default
+
+  def perform(user_id)
```

Comment:
This happens outside of a transaction so we can safely pass the actual `User` instance here and let GlobalID do its thing. Thus removing the need for `User.find(...)` below.